bugalink-backend-mock/src/data.ts

```diff
@@ -94,6 +94,67 @@ export const sampleTripRequests = [
     price: 1.5,
   },
 ];
+
+export const sampleUser = {
+  id: 1,
+  name: 'Paco',
+  lastName: 'Perez',
+  photo: '/assets/avatar.png',
+};
+
+export const IndividualRides = [
+  {
+    IndividualRideId:1,
```

Comment:
same, use either camelCase or snake_case for the API attributes but keep them consistent